plugins/inputs/sqlserver/sqlserver.go

```diff
@@ -422,9 +452,30 @@ BEGIN
 	EXEC sp_executesql @SqlStatement
 
 END
-ELSE
+ELSE IF @EngineEdition IN (2,3,4) /*Standard,Enterprise,Express*/
 BEGIN
 
+	DECLARE @MajorMinorVersion AS int = CAST(PARSENAME(CAST(SERVERPROPERTY('ProductVersion') as nvarchar),4) AS int) * 100 + CAST(PARSENAME(CAST(SERVERPROPERTY('ProductVersion') as nvarchar),3) AS int)
+
+	DECLARE @Columns as nvarchar(max) = ''
+	DECLARE @Tables as nvarchar(max) = ''
+
+	IF @MajorMinorVersion >= 1050 BEGIN 
+		/* [volume_mount_point] TRIMS trailing "\" which are not allowed in InfluxDB */
+		SET @Columns += N',LEFT(vs.[volume_mount_point], LEN(vs.[volume_mount_point])-(PATINDEX(''%[^\]%'',REVERSE([volume_mount_point]))-1)) AS [volume_mount_point]'
+		SET @Tables += N'CROSS APPLY sys.dm_os_volume_stats(vfs.[database_id], vfs.[file_id]) AS vs'
+	END
+		
+	IF @MajorMinorVersion > 1100 BEGIN
+		SET @Columns += N'
+,vfs.io_stall_queued_read_ms AS [rg_read_stall_ms] 
+,vfs.io_stall_queued_write_ms AS [rg_write_stall_ms]'
+	END
+	
+	/*Debug only*/
+	--PRINT @Columns
+	--PRINT @Tables
```

Comment:
Remove any debug lines, even commented, from from the final code.

Reply:
The idea was to keep those there to make debugging easier for anyone who wanted to check what was going, I would like to keep at least the last of them, which prints out the query that will be sent to SQL Server. (if that's against the standard/rules I will just remove them all)

Reply:
Normally, I wouldn't allow this in Go code.  In this case I'll leave it to @denzilribeiro

Reply:
These are never going to be triggerred unless uncommented so why leave it in the code? I am with @danielnelson  on this... if the intent is to run the query and debug elsewhere add it later. There is nothing from a config perspective that you can change to have this part triggered so I would remove.

Reply:
All the debug prints have been removed, a pair of them also in the v1 plugin